### PR TITLE
chore: remove disable on assessment out of order

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** workflow steps should work even it is out of order

JIRA: https://2u-internal.atlassian.net/browse/AU-1603

**NOTE** you can only re-arrange self and practice stage. ORA protect us from saving any other order.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by jansenk** [comment]
Looks like you need to update a test that's expecting reordered steps to mean that we can't use MFE views

**Review by mattcarter — APPROVED** [review]
approved
